### ISSUE TITLE
Outline - gets the scope rather than the variable name

### ISSUE THREAD

**Issue by DaveAppleton** [ether-camp/ethereum-sandbox#6]
This variable got listed in the outline as "private" and not "founder"

![image](https://cloud.githubusercontent.com/assets/1858302/22709688/a100a082-edb5-11e6-8a87-20f7509dc578.png)

![image](https://cloud.githubusercontent.com/assets/1858302/22709715/bbc34514-edb5-11e6-90d7-9f9ec5289845.png)

**Comment by DaveAppleton** [comment]
Another example

![image](https://cloud.githubusercontent.com/assets/1858302/22798868/9ff9bdce-ef3e-11e6-8010-9d122cdbca43.png)


![image](https://cloud.githubusercontent.com/assets/1858302/22798850/85bd9b38-ef3e-11e6-9d8a-af63ec51047d.png)

**Comment by romanman** [comment]
yep looks like we have to rewrite the outline support... lets outscope it for a while
